README.md

```diff
@@ -58,3 +60,27 @@ If you would like to start contributing to an issue or PR, please assign it to y
 ### Major Contributions
 
 We will publish a roadmap discussing long-term goals of the project very soon - stay tuned!
+
+# Sponsors
+
+## Development Sponsor
+Ongoing Flatcar Container Linux development is supported by our Development Sponsors.
+
+
+<img src="sponsors/kinvolk_400x400.jpg" width="100pt" alt="Kinvolk.io logo" title="Kinvolk.io logo" />
+
+[Kinvolk.io](https://www.kinvolk.io/)
+
+
+<img src="sponsors/gs-logo-on-dark.svg" width="450pt" alt="Giant Swarm logo" title="Giant Swarm logo">
+
+[Giant Swarm](https://www.giantswarm.io/)
```

Comment:
Maybe adding the sponsors in a table is better, as it allows the list to scale better without expanding the README vertically too much.
Also, should we just have links directly from the images? Sounds a bit redundant to have the labels with the names when the logos and titles already have that.